src/App.js

```diff
@@ -2,20 +2,21 @@ import { BrowserRouter as Router } from 'react-router-dom';
 import { Header } from './components/Header/Header';
 import { Navbar } from './components/Navbar/Navbar';
 import { AppContainer } from './App.styles';
-import { Text } from './components/Text/Text';
 import { Content } from './components/Content/Content';
-import { Footer } from './components/Footer/Footer';
 import { WebsiteFooter } from './components/Footer/WebsiteFooter';
+import { TicketProvider } from './TicketContext/TicketContext';
 function App() {
   return (
-    <AppContainer>
-      <Router>
-        <Navbar />
-        <Header />
-        <Content />
-        <WebsiteFooter />
-      </Router>
-    </AppContainer>
+    <TicketProvider>
```

Comment:
`TicketProvider` probably shouldn't be wrapped around the whole app as this will cause the navbar and footer components, which don't use the ticket context, to rerender when the state changes.